### DISCUSSION TITLE
Configure AWS Amplify - Requires manual clicking in browser window

### DISCUSSION THREAD

**Discussion by rohit3d2003** [codemagic-ci-cd/codemagic-docs#2136] [Q&A]
We have a flutter app that depends on AWS amplify backend. This requires installing amplify followed by configuring it via aws key/secret. Unfortunately the configuration step opens a browser window to confirm authentication or copy cli key. I am not sure how to carry this particular step via yaml file or any alternate option like rdp etc. Appreciate any help on this.

**Comment by himesh-cm** [comment]
Hi @rohit3d2003,
You should be able to use headless authentication as shown in the official docs [here](https://docs.amplify.aws/cli/usage/headless/).
Can provide us with the build ID? i can further look into the same.

**Reply by himesh-cm** [reply]
I have deleted the comment. Please do not share screenshots of your internal dashboard. I have the build ID and will investigate further. I will keep you posted.

**Reply by himesh-cm** [reply]
Hi @rohit3d2003, Please note that you should contact our team through the chat widget in the UI for a quicker response, as your team is on a paid plan.

Regarding your query, It looks like a known issue from the error

`auth headless is missing the following inputParameters googleAppIdUserPool, googleAppSecretUserPool`

Please refer to the GitHub issue here - https://github.com/aws-amplify/amplify-cli/issues/9176
and possible resolution here - https://github.com/aws-amplify/amplify-js/issues/7194#issuecomment-727243245

Let me know, in case of any queries.

**Comment by himesh-cm** [comment]
Hi @rohit3d2003,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
